cart/forms.py

```diff
@@ -0,0 +1,8 @@
+from django import forms
+from .models import Cart
+
+class TickForm(forms.ModelForm):
+    tick = forms.BooleanField(label = 'ok')
+    model = Cart
+
```

Comment:
```class CartForm(forms.ModelForm):
    class Meta:
        model = Cart
        fields = ['product', 'quantity']
        widgets = {
            'product': forms.HiddenInput(),
        }
```

Reply:
Name the form according to the model that's why i changed this to CartForm, here I have changed the widget of product field to hidden since the product field should be auto-populated using `initail` parameter in the view.